web/src/main/webapp/js/graphhopper/GHRequest.js

```diff
@@ -70,7 +71,7 @@ GHRequest.prototype.init = function (params) {
         else if (val === "true")
             val = true;
 
-        if (key === "point" || key === "mathRandom" || key === "do_zoom" || key === "layer")
+        if (key === "point" || key === "mathRandom" || key === "do_zoom" || key === "layer" || key === "useMiles")
```

Comment:
Well, inside JavaScript and Java we should aim for camelCase variable naming as it is standard there (not 100% sure for JS). But outside of code (configuration, json, URL) we should use under score to make it consistent accross languages and data formats. That was not clear from the beginning so there might be still inconsistency in JS, hopefully not anymore in Java core. See #682